resources/views/dashboard/controllers/events/view.blade.php

```diff
@@ -60,7 +60,7 @@
                                 <th scope="col">Controller</th>
                                 <th scope="col">Availability</th>
                                 @if(Auth::user()->isAbleTo('events'))
-                                <th scope="col">Actions</th>
+                                <th scope="col">Actions   </th>
```

Comment:
Why the space here? If we need white space, it would be more consistent to add padding or a margin using bootstrap (https://stackoverflow.com/questions/32233489/does-bootstrap-have-builtin-padding-and-margin-classes)

Reply:
Nice catch - forgot to remove this